src/libraries/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -7,12 +7,12 @@ internal static partial class Interop
     internal static partial class Libraries
     {
         // Shims
-        internal const string SystemNative = "System.Native";
```

Comment:
looks like there's another file which mentions `System.Native`: https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/CoreFx.Private.TestUtilities/Interop/Interop.Libraries.cs

Reply:
That is just a test file and works with and without `lib` even in this file. The only reason of adding `lib` here is because it is a product code, and without lib it will incur a probe step: https://github.com/dotnet/runtime/pull/33236#issuecomment-595276121.